src/components/interfaces/HMI_API.xml

```diff
@@ -6005,6 +6051,9 @@
     <param name="turnSignal" type="Boolean" mandatory="false">
         <description>See TurnSignal</description>
     </param>
+    <param name="gearStatus" type="Boolean" mandatory="false">
+      <description>See GearStatus</description>
+    </param>
     <param name="prndl" type="Boolean" mandatory="false">
       <description>See PRNDL</description>
```

Comment:
Should add a note for all `prndl` params stating that they are deprecated in the MOBILE API and that the param is covered by `gearStatus`